db/migrate/20171118114518_create_delayed_jobs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateDelayedJobs < ActiveRecord::Migration[5.1]
+  def self.up
+    create_table :delayed_jobs, force: true do |table|
+      table.integer :priority, default: 0, null: false # Allows some jobs to jump to the front of the queue
+      table.integer :attempts, default: 0, null: false # Provides for retries, but still fail eventually.
+      table.text :handler,                 null: false # YAML-encoded string of the object that will do work
```

Comment:
Don't align the `null: false` to the other lines. Just keep it as one space from the comma